boilerplate/openshift/golang-osd-operator/standard.mk

```diff
@@ -125,3 +115,44 @@ yaml-validate: python-venv
 .PHONY: validate-olm-deploy-yaml
 validate-olm-deploy-yaml: python-venv
 	${PYTHON} ${CONVENTION_DIR}/validate-yaml.py $(shell git ls-files 'deploy/*.yaml' 'deploy/*.yml')
+
+######################
+# Targets used by prow
+######################
+
+# validate: Ensure code generation has not been forgotten; and ensure
+# generated and boilerplate code has not been modified.
+# TODO
```

Comment:
Missing `TODO` content?

Reply:
This is saying the guts of the whole target still need to be done.

Reply:
I added more words here, since I was rebasing.